build.gradle

```diff
@@ -57,42 +59,65 @@ subprojects {
             force "commons-codec:commons-codec:1.15"
             force "org.apache.httpcomponents:httpcore:4.4.15"
             force "jakarta.annotation:jakarta.annotation-api:2.1.1"
+            force "org.javassist:javassist:3.25.0-GA"
+            force "org.apache.httpcomponents:httpclient:4.5.14"
             // Force spotless depending on newer version of guava due to CVE-2023-2976. Remove after spotless upgrades.
             force "com.google.guava:guava:32.1.2-jre"
             // Force opensearch transport-netty4-client depending on newer version of netty-handler due to CVE-2023-34462.
             // Remove after upgrading the compiled opensearch version.
             force "io.netty:netty-handler:4.1.94.Final"
         }
     }
-
+    
     dependencies {
         implementation "net.jodah:failsafe:2.4.0"
         implementation "com.google.guava:guava:32.1.2-jre"
         implementation "org.apache.commons:commons-lang3:3.9"
         implementation "commons-io:commons-io:2.7"
+        implementation "commons-logging:commons-logging:1.2"
+        implementation "commons-codec:commons-codec:1.15"
+        
         /******************** Hacking transient dependencies for OCI SDK ********************/
         // TODO: For some odd reason it seems that the OpenSearch gradle plugin overrides resolution of implementation
         // configuration which is why this hack is needed so we can get the transient dependencies packaged properly
 
 
+        implementation("com.oracle.oci.sdk:oci-java-sdk-common-httpclient:${sdk_version}")
         implementation("com.oracle.oci.sdk:oci-java-sdk-common-httpclient-jersey:${sdk_version}")
-        compileClasspath("com.oracle.oci.sdk:oci-java-sdk-common-httpclient-jersey:${sdk_version}")
-        testCompileClasspath("com.oracle.oci.sdk:oci-java-sdk-common-httpclient-jersey:${sdk_version}")
-        runtimeClasspath("com.oracle.oci.sdk:oci-java-sdk-common-httpclient-jersey:${sdk_version}")
-        testRuntimeClasspath("com.oracle.oci.sdk:oci-java-sdk-common-httpclient-jersey:${sdk_version}")
-
         implementation("com.oracle.oci.sdk:oci-java-sdk-objectstorage:${sdk_version}")
-        compileClasspath("com.oracle.oci.sdk:oci-java-sdk-objectstorage:${sdk_version}")
-        testCompileClasspath("com.oracle.oci.sdk:oci-java-sdk-objectstorage:${sdk_version}")
-        runtimeClasspath("com.oracle.oci.sdk:oci-java-sdk-objectstorage:${sdk_version}")
-        testRuntimeClasspath("com.oracle.oci.sdk:oci-java-sdk-objectstorage:${sdk_version}")
-
-        runtimeClasspath("org.slf4j:slf4j-api") {
-            version {
-                strictly "1.7.33"
-            }
-        }
-        testRuntimeClasspath("org.slf4j:slf4j-api") {
+        implementation("com.oracle.oci.sdk:oci-java-sdk-common:${sdk_version}")
```

Comment:
Those are not new dependencies, all of them were brought in transitively

Reply:
Got it, thanks @reta approved.

Reply:
@samuel-oci do you mind approving the workflow? seems like few checks need to be run, thank you

Reply:
done

Reply:
Apologies, @samuel-oci may I ask you to approve one more time (fixed build issue), thank you